types/index.esm.d.ts

```diff
@@ -2418,15 +2418,10 @@ export interface TooltipOptions<TParsedData> extends CoreInteractionOptions {
 	 */
 	footerAlign: TextAlign;
 	/**
-	 * Padding to add on left and right of tooltip.
+	 * Padding to add to the tooltip
 	 * @default 6
 	 */
-	xPadding: number;
-	/**
-	 * Padding to add on top and bottom of tooltip.
-	 * @default 6
-	 */
-	yPadding: number;
+	padding: number | ChartArea;
```

Comment:
Not related to this PR really, but looks like the `ChartArea` could use more generic name.